candidate-testing.js

```diff
@@ -3,31 +3,37 @@ const input = require('readline-sync');
 // TODO 2: modify your quiz app to ask 5 questions //
 
 // TODO 1.1a: Define candidateName // 
-let candidateName;
+let candidateName = '';
 // TODO 1.2a: Define question, correctAnswer, and candidateAnswer //
-let question;
-let correctAnswer;
+let question  = "Who was the first American woman in space? ";
+let correctAnswer = "Sally Ride";
 let candidateAnswer;
```

Comment:
I think you meant to initialize this candidateAnswer as a string instead of the candidateAnswers on line 13. ;)

Reply:
Chad, it's good. This is a pass on canvas. If you want to see a green check mark you need to:
- go back to this : "let candidateAnswer;"  in line 10 and initialize it with and empty string (""). That's the failing test in github.